### PR TITLE
(XW-1979) Support search shortcut in new nav

### PR DESCRIPTION
**Pull request by Grunny**
https://wikia-inc.atlassian.net/browse/XW-1979

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [AddDefaultShortcuts.js#L5](https://github.com/Wikia/app/blob/34db2f023ea4e6e95e406b9278fbb0e4a919b08a/extensions/wikia/GlobalShortcuts/scripts/AddDefaultShortcuts.js#L5): Function has a complexity of 12 which is greater than 10 authorized. [rule](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AFunctionComplexity)

File: extensions/wikia/GlobalShortcuts/scripts/AddDefaultShortcuts.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ require(['GlobalShortcuts', 'PageActions', 'mw', 'wikia.loader'],
 						id: 'wikia:Search',
 						caption: mw.message('global-shortcuts-caption-search-for-a-page').plain(),
 						fn: function () {
-							$('#searchInput')[0].focus();
+							$('.wds-global-navigation').find('.wds-global-navigation__search-input')[0].focus();
```
Comment (rogatty) [inline extensions/wikia/GlobalShortcuts/scripts/AddDefaultShortcuts.js]: @kvas-damian added `id="globalNavigation"` in https://github.com/Wikia/app/pull/11281/files#diff-f850972c6a894a3d071e2a573d2aa230R2. I think we should do the same here.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 6 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MINOR [DesignSystemGlobalNavigation_index.php#L1](https://github.com/Wikia/app/blob/e0b93703227fc1202c49fe846f4bf067cf6c79b9/extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php#L1): Define a constant instead of duplicating this literal "fandom_overview" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
2. MINOR [DesignSystemGlobalNavigation_index.php#L3](https://github.com/Wikia/app/blob/e0b93703227fc1202c49fe846f4bf067cf6c79b9/extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php#L3): Define a constant instead of duplicating this literal "model" 5 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
3. MINOR [DesignSystemGlobalNavigation_index.php#L3](https://github.com/Wikia/app/blob/e0b93703227fc1202c49fe846f4bf067cf6c79b9/extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php#L3): Define a constant instead of duplicating this literal "DesignSystemGlobalNavigationService" 5 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
4. MINOR [DesignSystemGlobalNavigation_index.php#L6](https://github.com/Wikia/app/blob/e0b93703227fc1202c49fe846f4bf067cf6c79b9/extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php#L6): Define a constant instead of duplicating this literal "links" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
5. MINOR [DesignSystemGlobalNavigation_index.php#L21](https://github.com/Wikia/app/blob/e0b93703227fc1202c49fe846f4bf067cf6c79b9/extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php#L21): Define a constant instead of duplicating this literal "search" 4 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
6. MINOR [DesignSystemGlobalNavigation_index.php#L21](https://github.com/Wikia/app/blob/e0b93703227fc1202c49fe846f4bf067cf6c79b9/extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigation_index.php#L21): Define a constant instead of duplicating this literal "module" 4 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)

**Comment by rogatty** [comment]
LGTM

**Comment by kvas-damian** [comment]
LGTM

**Comment by Warkot** [comment]
LGTM

**Pull request merged**